Enterprise-CMCS/macpro-mdct-mfp
# [CMDCT-159] Target Populations added to SAR Form Creation

### Description

Does this work? Am I crazy? Why not both? You decide.
Prepare your mental fortitude and if you can make it through the madness I wrote today, then Im glad someone was able to.

https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/19439679/782be747-561e-48ed-bf35-1ee17db58a26


### Related ticket(s)


CMDCT-159

---
### How to test

Create a work plan
Add a transition benchmark, if you want,
Create a SAR and see its in the modal!
Edit a SAR on the dashboard and see its in the modal!

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
